tests/foreman/cli/test_organization.py

```diff
@@ -1693,3 +1694,32 @@ def test_positive_info_by_name(self):
         org = make_org()
         result = Org.info({'name': org['name']})
         self.assertEqual(org['id'], result['id'])
+
+    @tier1
+    @skip_if_bug_open('bugzilla', 1418412)
+    def test_positive_multibyte_latin1_org_names(self):
+        """Hammer Multibyte and Latin-1 Org names break list pagination
+
+        :id: 4fa0afe7-6d0a-4c3e-a0fc-4ecb95c50fc9
+
+        :BZ: 1418412
+
+        :expectedresults: Multibyte and latin1 names need to be
+            displayed with consistent spacing
+        """
+        org_names = valid_org_names_list()
+        for org in org_names:
+            make_org({'name': org})
+        org_list = Org.list(output_format='table').remove('')
+        for org in org_names:
+            Org.delete({'name': org})
+        if org_list:
```

Comment:
This check is basically wrong. In case org_list is empty - no steps  will be performed and test will show us false positive. And, unfortunately, that's what really happened - `.remove('')` returned you `None`, `if org_list:` was skipped as org_list is None, test showed you 'PASSED' but it verified nothing.

Use assertions instead, `self.assertGreaterEqual(len(org_list), len(org_names))` in this case - this way you'll ensure we have enough orgs in output and fail the test otherwise. And, as a bonus, it will reduce unnecessary indentation :)

I'd also think of `self.assertIn` to ensure concrete orgs are present in output - as otherwise (lets say orgs with special characters appeared on the next page) the test won't really verify bugzilla's bug.